bin/zk

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-tmux new-window -c "${ZK_PATH}" "bash -l -c -i vim" \; split-window -h "bash -i -l "zks""
+tmux new-window -c "${ZK_PATH}" "bash -l -c -i nvim" \; split-window -h "bash -i -l "zks""
```

Comment:
I can remove these nvims again, but maybe people new to the nvim / vim world tend to choose nvim? 🤔 I kept them for now, since the `zkn` script  calls nvim already, so now it should "all use the same editor" 😅